Integrations/integration-PhishMe.yml

```diff
@@ -182,31 +241,39 @@ script:
         var md = "## PhishMe email Reputation for: " + sender_name + "\n";
         var dbotScore = 0;
         var threatArray = [];
-        if (threats.length !== 0) {
+        var threshold = params.emailThreshold;
+        var indicatorFnd = false;
+        if (threats.length && threats.length !== 0) {
             ec.Email = {
                 'Data': sender_name,
                 'Malicious': {'Vendor': 'PhishMe', 'Description': 'Match found in PhishMe database'}
             };
             ec.Account = {
-                'Account.Email(val.Address && val.Address === obj.Address)': {
+                'Email': {
```

Comment:
why the dt was removed?

Reply:
Had issues with it - for some reason the dt statement was set into the context as is, rather than executing the dt. I'll try to see if I can fix that and return the dt into the key.